ultraplot/tests/test_plot.py

```diff
@@ -225,3 +225,41 @@ def test_quiver_discrete_colors():
     ax.quiver(X - 2, Y, U, V, C)
     ax.quiver(X - 3, Y, U, V, color="red", infer_rgb=True)
     return fig
+
+
+def test_setting_log_with_rc():
```

Comment:
This test is testing the wrong thing. You need to check the tick label formatters, not the scale of the axis itself.

Reply:
Thanks I updated the test. Have a look